certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/common/model/CertificatesRequest.java

```diff
@@ -26,6 +27,9 @@ public class CertificatesRequest {
   HsaId careUnitId;
```

Comment:
@mhornfeldt need to verify with what is sent from IT and req, if this needs to be filtered on also or if issued is enough.